go.mod

```diff
@@ -1,23 +1,23 @@
 module github.com/openshift/cluster-dns-operator
 
-go 1.19
+go 1.20
```

Comment:
should be `go 1.21` ?

Reply:
Thanks. I saw your PR to update this and other files and will make the updates here.